src/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List Structure</title>
+    <link rel="stylesheet" href="./style.css">
+    <script src="https://kit.fontawesome.com/e520ead38a.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet"/>
+</head>
+
+<body>
+    <section class="main">
+        <div class="main-container">
+            <ul id="list-container">
+                <li>Today's To Do <i class="fa-solid fa-arrows-rotate"></i></li>
+                <hr>
+                <li><input type="text" class="list-add" placeholder="Add your list..."> <i class="fa-solid fa-arrow-turn-down"></i></li>
+                <hr>
+            </ul>
+            <ul class="clear-list">
+                <li>Clear all completed</li>
+            </ul>
+        </div>
+    </section></body>
+

```

Comment:
- [ ] kindly please remove unusable Blanck space for indentations without a reason 👍 .